src/iotjs_string_ext.c

```diff
@@ -48,9 +48,9 @@ static const jerry_length_t magic_string_lengths[] = {
 //
 // declare strings table
 //
-static const jerry_char_ptr_t magic_string_items[] = {
+static const jerry_char_t *magic_string_items[] = {
 #define MAGICSTR_EX_DEF(NAME, STRING) \
-  (const jerry_char_ptr_t) jerry_magic_string_ex_##NAME,
+  (const jerry_char_t *)jerry_magic_string_ex_##NAME,
```

Comment:
Space is missing after the closing bracket.

Reply:
Actually IoT.js style must be applied here (not JerryScript style), so the space is not needed.